src/Workspaces/Core/Portable/LanguageServices/SyntaxFactsService/ISyntaxFactsService.cs

```diff
@@ -286,10 +287,14 @@ internal interface ISyntaxFactsService : ILanguageService
         bool IsTopLevelNodeWithMembers(SyntaxNode node);
         bool HasIncompleteParentMember(SyntaxNode node);
 
+        bool IsPureBlock(SyntaxNode node);
         bool IsExecutableBlock(SyntaxNode node);
```

Comment:
good to doc these.

Reply:
➡️ Fixed in 9568995bdf98d9f7c93970d0f1d25839f4c1e8a0